packages/node_modules/@ciscospark/spark-core/src/interceptors/redirect.js

```diff
@@ -50,6 +50,7 @@ export default class RedirectInterceptor extends Interceptor {
    * @returns {Object}
    */
   onResponse(options, response) {
+    /* eslint-disable no-else-return */
```

Comment:
Is this eslint-disable necessary? Your changes don't seem to violate this rule.

Reply:
Yes, it gives an eslint error for no-else-return otherwise.